gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -272,6 +278,7 @@
   public static final String DATA_PUBLISHER_FINAL_DIR = DATA_PUBLISHER_PREFIX + ".final.dir";
   public static final String DATA_PUBLISHER_REPLACE_FINAL_DIR = DATA_PUBLISHER_PREFIX + ".replace.final.dir";
   public static final String DATA_PUBLISHER_FINAL_NAME = DATA_PUBLISHER_PREFIX + ".final.name";
+  public static final String DATA_PUBLISHER_GROUP_NAME = DATA_PUBLISHER_PREFIX + ".group.name";
```

Comment:
OK, `DATASET_URN_KEY` is used to specify the URN of a dataset processed by a `WorkUnit`, whereas  `DATA_PUBLISHER_GROUP_NAME` is used to specify the own group of the final data publisher dir. I changed `DATA_PUBLISHER_GROUP_NAME` to `DATA_PUBLISHER_FINAL_DIR_GROUP`.